notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -10417,18 +10419,16 @@ https://en.wikipedia.org/wiki/Species_II
 :: Genre
 sci-fi
 
-:: Choice Themes
-human-alien hybrid [The main novelty of the story was Patrick becoming a hyper-sexual, human-alien amalgam after getting exposed to alien DNA. Eve was also half-human and half-alien.]
```

Comment:
This is just my musings: I've taken to use present tense when I write about the story, I.e, "the main novelty of the story is..." because the story continues to exist. I use past tense when I write about things inside the story, because (I guess) I mentally insert a "we heard that..." before any observation. Or maybe I see the conditions inside the story to have ceased existing when I stopped reading. Anyway, this has seemed like a good rule of thumb for some time even if I'm sure I violate it frequently.

Reply:
I've been following the rule of thumb you lay out there for the most part, actually. Why I didn't follow it here is anybody's guess :-| I'll update it to read 

"The main novelty of the story concerns Patrick becoming a hyper-sexual, human-alien amalgam after getting exposed to alien DNA. Eve was also half-human and half-alien."

after I figure out how to sync the dev and master branches.